internal/ocm/ocm.go

```diff
@@ -383,3 +391,25 @@ func (csc *clusterServiceClient) ListBreakGlassCredentials(clusterInternalID Int
 	}
 	return BreakGlassCredentialListIterator{request: breakGlassCredentialsListRequest}
 }
+
+func (csc *clusterServiceClient) GetVersion(ctx context.Context, versionName string) (*cmv1.Version, error) {
+	client := cmv1.NewVersionClient(csc.conn, versionName)
+
+	resp, err := client.Get().SendContext(ctx)
+	if err != nil {
+		return nil, err
+	}
+	version, ok := resp.GetBody()
+	if !ok {
+		return nil, fmt.Errorf("empty response body")
+	}
+	return version, nil
+}
+
+func (csc *clusterServiceClient) ListVersions(searchExpression string) VersionsListIterator {
+	versionsListRequest := csc.conn.ClustersMgmt().V1().Versions().List()
```

Comment:
Same as above.  I guess I overlooked this on the first go round.
```suggestion
	versionsListRequest := csc.conn.AroHCP().V1alpha1().Versions().List()
```